components-dist/components/ConversationView/index.d.ts

```
@@ -0,0 +1,18 @@
+import { CopilotConfigType } from '../../contexts/types';
+import { OnSendCallbackType } from '../../widgets/copilot/components/ChatInput';
+
+export type ConversationViewProps = {
+    messages: Map<string, any>;
+    isSending?: boolean;
+    scrollContainerRef?: React.RefObject<HTMLDivElement> | null;
+    isMessagesLoading?: boolean;
+    avoidAutoplay?: () => void | null;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)